### PR TITLE
Allow adding but not editing for some endpoints

### PR DESCRIPTION
**Pull request by jacobwindsor**
dcp-752

Allows importer to continue importing while submission is in PENDING or METADATA_VALID.

**Please Check**
- All endpoints that are used by importing have been changed
- No extra endpoints that aren't used by importing have been changed

### CODE REVIEW

File: src/test/java/org/humancellatlas/ingest/submission/SubmissionEnvelopeTest.java [hunk 1]
```diff
@@ -155,6 +152,48 @@ public void testIsEditable() {
         });
     }
 
+    @Test
+    public void testCanAddTo() {
```
Comment (MightyAx) [inline src/test/java/org/humancellatlas/ingest/submission/SubmissionEnvelopeTest.java:156]: I have a commit here to make the test parameterised

**Review by MightyAx — CHANGES_REQUESTED** [review]

**Review by MightyAx — APPROVED** [review]

**Comment by jacobwindsor** [comment]
Oooh I always forget about parameterized tests. Thanks @MightyAx

**Comment by MightyAx** [comment]
> Oooh I always forget about parameterized tests. Thanks @MightyAx

JUnit doesn't make it easy, you can only parameters one input, and you can only parameterise basic types and enums

**Comment by jacobwindsor** [comment]
@MightyAx and you added endpoint tests! Amazing, you're a star

**Comment by jacobwindsor** [comment]
Merging as is small change and we're short on devs

**Pull request merged**